pages.es/linux/gedit.md

```diff
@@ -0,0 +1,20 @@
+# gedit
+
+> Editor de texto del proyecto GNOME.
+> Más información: <https://help.gnome.org/users/gedit/stable/>.
+
+- Abrir un fichero de texto:
```

Comment:
For Romance languages, the present indicative fits better.
```suggestion
- Abre un archivo de texto:
```
We use `archivo` over `fichero`.